package.json

```diff
@@ -37,7 +36,7 @@
 		"typescript": "^5.5.3"
 	},
 	"peerDependencies": {
-		"@astrojs/db": "^0.12.0",
+		"@astrojs/db": "0.12 || 0.13",
```

Comment:
```suggestion
		"@astrojs/db": ">= 0.12",
```

Reply:
I was thinking on it and finally decide the limited version, but I also prefer keep it open to future version